tests/conftest.py

```diff
@@ -1,6 +1,6 @@
 import builtins
 import pytest
-from tools import DummyShell, sp
+from tests.tools import DummyShell, sp
```

Comment:
Well tests seem to pass but don't understand why this would work. Shouldn't the import be relative to this conftest.py file?